content/faq/en.md

```diff
@@ -44,7 +45,7 @@ We focus most heavily on **Javascript-related** topics, but we may also explore
 
 ### When is the next meetup?
 
-Meetups usually happen **every last Wednesday of the month**.  
+Meetups usually happen **every last Wednesday of the month**.
```

Comment:
Not sure this was intentional. 2 spaces at the end of line is markdown for `<br>`.

Reply:
frrrrrrr
was my prettier ...